### PR TITLE
Update esm-samples to 4.26

### PR DESCRIPTION
**Pull request by andygup**
Updated all samples. No notable changes.

### CODE REVIEW

**Review by dasa — CHANGES_REQUESTED** [review]
Update to latest Rollup and remove `experimentalDeepDynamicChunkOptimization`.

Ref: https://github.com/rollup/rollup/releases/tag/v3.17.0

**Comment by andygup** [comment]
Good catch. Updated.

**Review by odoe — APPROVED** [review]
Looking good

**Comment by dasa** [comment]
> Good catch. Updated.

The first known issue could be removed too: https://github.com/Esri/jsapi-resources/tree/update-esm-4.26/esm-samples/rollup#known-issues

File: .github/ISSUE_TEMPLATE/issue-report.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 name: Issue
 description: Report an issue with one of the samples in this repository. 
-labels: ["needs triage"]
```
Comment (dasa) [inline .github/ISSUE_TEMPLATE/issue-report.yml:3]: Was this change intentional, as well as the deletion of `SUPPORT.md`?

File: esm-samples/jsapi-custom-workers/package.json [hunk 2]
```diff
@@ -12,7 +10,7 @@
     "html-webpack-plugin": "^5.5.0",
     "mini-css-extract-plugin": "^2.7.2",
     "npm-run-all": "^4.1.5",
-    "rollup": "^3.10.1",
+    "rollup": "^3.16.0",
```
Comment (dasa) [inline esm-samples/jsapi-custom-workers/package.json:13]: Update?

File: esm-samples/jsapi-node/package.json [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
     "rollup-plugin-copy": "^3.4.0",
     "rollup-plugin-delete": "^2.0.0"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^22.0.0",
-    "@rollup/plugin-node-resolve": "^13.3.0",
-    "rollup": "^2.75.6"
+    "@rollup/plugin-commonjs": "^24.0.1",
+    "@rollup/plugin-node-resolve": "^15.0.1",
+    "rollup": "^3.16.0"
```
Comment (dasa) [inline esm-samples/jsapi-node/package.json:13]: Update?

File: .github/ISSUE_TEMPLATE/issue-report.yml (hunk 1 above)
Comment (andygup) [reply .github/ISSUE_TEMPLATE/issue-report.yml:3]: Yes, that was intentional. We currently aren't using `SUPPORT.md` as a template, and @odoe had suggested removing the triage tag.

File: esm-samples/jsapi-custom-workers/package.json (hunk 2 above)
Comment (andygup) [reply esm-samples/jsapi-custom-workers/package.json:13]: Oh, I was doing before/after testing and accidentally checked in the wrong version. I'll update and retest for this sample and jsapi-node.

**Review by dasa — APPROVED** [review]
👍🏻

**Comment by andygup** [comment]
@ignore the last github action error on this PR. I cycled the action to verify that a small update worked.

**Comment by andygup** [comment]
Github action fix was applied and build was successful. Something changed in the public github that invalidated the pattern we used to stash and pop. Merging.
